frappe/public/js/frappe/form/controls/base_input.js

```diff
@@ -95,6 +95,7 @@ frappe.ui.form.ControlInput = frappe.ui.form.Control.extend({
 					if (me.disp_area) {
 						me.set_disp_area(me.value);
 						$(me.disp_area).toggle(true);
+						make_input();
```

Comment:
@shariquerik This change has some side effects.

Like read-only "Code" control should not show "Expand" button but it does after this change.

**Expected** (without this change):
<img width="544" alt="Screenshot 2021-02-22 at 6 12 09 PM" src="https://user-images.githubusercontent.com/13928957/108710326-eaee3f00-7539-11eb-8e5a-6d26916a400e.png">

**With this change**:
<img width="570" alt="Screenshot 2021-02-22 at 6 12 47 PM" src="https://user-images.githubusercontent.com/13928957/108710380-fa6d8800-7539-11eb-8904-5e1002656bd1.png">


Please check for other controls as well.

Reply:
Ok, I will check.

Reply:
I can add condition: 

`if (me.df.options == 'Phone') {
    make_input();
}`

it will work fine. Can you suggest anything better?